curp/build.rs

```diff
@@ -6,7 +6,6 @@ fn main() {
         )
         .compile(
             &[
-                "./proto/common/src/message.proto",
```

Comment:
Why do you remove this `message.proto` file

Reply:
It seems that `curp-command.proto` could be merged into `message.proto`. I choose to keep `curp-command.proto`.